### ISSUE TITLE
Clairification on what value 'delta' will have in the wiringPiISR callback.  

### ISSUE THREAD

**Issue by TheNotary** [WiringPi/WiringPi-Node#65]
According to the docs on [interrupts](https://github.com/WiringPi/WiringPi-Node/blob/master/DOCUMENTATION.md#interrupts) the callback to wiringPiISR has a parameter (it is named delta in the example).  It seems to spit out random integers when I print those to the screen.  What do those ints represent?

**Comment by r1ch** [comment]
Yes - it's a bit of a mystery! Looking the C code that's backing the function we've got:
unsigned int now = ::micros();
data->delta = now - data->previous_timestamp;
data->previous_timestamp = now;

It's an (educated!) guess but I think it's the microseconds elapsed since the last interrupt.

Still - I was led here by Google with the same question so... any other answers appreciated!
